Cargo.toml

```diff
@@ -124,13 +125,7 @@ default = ["std", "use-libc-auxv"]
 std = ["io-lifetimes"]
 
 # This is used in the port of std to rustix.
-rustc-dep-of-std = [
-    "core",
-    "alloc",
-    "compiler_builtins",
-    "linux-raw-sys/rustc-dep-of-std",
-    "bitflags/rustc-dep-of-std",
-]
+rustc-dep-of-std = ["core", "alloc", "compiler_builtins", "linux-raw-sys/rustc-dep-of-std", "bitflags/rustc-dep-of-std"]
```

Comment:
The resulting line here is 120 columns wide, which is wider than Rust's typical line length limit, making it awkward to read. Could you revert this formatting change?

Reply:
`cargo add` just does that for some reason. Fixed!